README.rst

```diff
@@ -174,3 +181,7 @@ for the complete license.
 .. |coverage| image:: https://codecov.io/github/borgbackup/borg/coverage.svg?branch=master
         :alt: Test Coverage
         :target: https://codecov.io/github/borgbackup/borg?branch=master
+
+.. |screencast| image:: https://asciinema.org/a/28691.png
+        :alt: BorgBackup Installation and Basic Usage
+        :target: https://asciinema.org/a/28691?autoplay=1&speed=2
```

Comment:
i'ld rather keep readme.rst as is for now.

esp. you shouldn't do frontpage redesign in a PR titled "markup fixes".